EIPS/eip-6122.md

```diff
@@ -0,0 +1,189 @@
+---
+eip: 6122
+title: Forkid checks based on timestamps
+author: Marius van der Wijden (@MariusVanDerWijden)
+description: Modifies the forkid checks to work with timestamps and block numbers
+discussions-to: https://ethereum-magicians.org/t/eip-6122-forkid-checks-based-on-timestamps/12130
+status: Draft
+type: Standards Track
+category: Networking
+created: 2022-12-13
+requires: 2124
+---
+
+## Abstract
+
+[EIP-2124](eip-2124.md) proposed a way of identifying nodes on the p2p network based on their chain configuration via the forkid parameter. 
+It allows nodes to cut incompatible nodes off quickly which makes the P2P network more reliable.
+After the merge forks are scheduled by blocktime instead of blocknumber. This EIP updates the Forkid calculation with blocktime.
+
+## Motivation
+
+While the execution layer scheduled forks by blocknumber, the consensus layer scheduled forks by slot number. The slot number is a time based measurement. In order to schedule forks at the same time on the consensus and execution layer, the execution layer is forced to also schedule forks by timestamp after the merge.
```

Comment:
The execution layer had a consensus -- proof-of-work. I think it's clearer to call that out explicitly 

```suggestion
While in proof-of-work forks were scheduled by block number, the proof-of-stake consensus layer schedules forks by slot number. The slot number is a time based measurement. In order to schedule forks at the same time on the consensus and execution layer, the execution layer is forced to also schedule forks by timestamp after the merge.
```